.storybook/webpack.config.js

```diff
@@ -1,13 +1,25 @@
 const path = require( 'path' );
 const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
+const mainConfig = require( '../webpack.config' );
+const mapValues = require( 'lodash/mapValues' );
 
 module.exports = async ( { config } ) => {
+	// Site Kit loads its API packages as externals,
+	// so we need to convert those to aliases for Storybook to be able to resolve them.
+	const siteKitPackageAliases = mapValues(
+		mainConfig.siteKitExternals,
+		( [ global, api ] ) => {
+			return path.resolve( `assets/js/${ global }-${ api }.js` );
+		}
+	);
+
 	config.resolve = {
 		...config.resolve,
 		alias: {
 			...config.resolve.alias,
-			'@wordpress/api-fetch$': path.resolve( __dirname, 'wp-api-fetch-mock.js' ),
```

Comment:
This shouldn't have been removed _yet_ (I'll put it back), but it should be removed.

This is a legacy mock that resolves datapoint requests to `global.googlesitekit.modules[module][datapoint]`. 

AFAIK, it was only used for Analytics, but going forward fetches should be "pre-resolved" with `receive*` actions or, if necessary, use a real mock on `fetch` (see added `fetch-mock` since we can't use Jest mocks in Storybook).